codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -120,65 +108,45 @@ var Worksheet = React.createClass({
             return false;
         }.bind(this));
 
-        //toggle search bar - B
-        Mousetrap.bind(['shift+b'], function(e){
+        // Show/hide web terminal
+        Mousetrap.bind(['shift+c'], function(e){
             this.toggleActionBar();
         }.bind(this));
 
-         Mousetrap.bind(['c'], function(e){
-                this.showActionBar();
-                this.setState({activeComponent: 'action'});
-                $('#command_line').terminal().focus();
+        // Focus on web terminal (action bar)
+        Mousetrap.bind(['c'], function(e){
+            this.showActionBar();
+            this.setState({activeComponent: 'action'});
+            $('#command_line').terminal().focus();
         }.bind(this));
 
-        //toggle raw - F
-        Mousetrap.bind(['shift+f'], function(e){
-            this.toggleRawMode();
+        // Toggle edit mode
+        Mousetrap.bind(['e'], function(e){
+            this.toggleEditMode();
             return false;
         }.bind(this));
-
-        //turn on edit mode or turn it off - E
-        Mousetrap.bind(['shift+e'], function(e){
-            this.toggleEditing();
-        }.bind(this));
-    },
-    toggleEditingText: function(arg){
-        this.setState({editingText: arg});
     },
-    toggleEditing: function(arg){
-        if(typeof(arg) !== 'undefined'){
-            this.setState({editMode:arg});
-        }else {
-          this.setState({editMode:!this.state.editMode});
-        }
-    },
-    toggleRawMode: function(val){
-        if(typeof(val)=='undefined'){
-            if(this.state.rawMode){
-                ///TODO grab val the react way
-                ws_obj.state.raw = $("#raw-textarea").val().split('\n');
-                if(this.canEdit()){
-                    this.saveAndUpdateWorksheet(true);
-                }
+    toggleEditMode: function(editMode) {
+        if (typeof(editMode) == 'undefined')
```

Comment:
nit: I know this was not added in this diff, and not sure what our style guidelines are (maybe we can adopt the Google JavaScript Style guide somewhere in the wiki), but I think these comparisons should usually be `===`.